app/components/Card/CardBack.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Card as ChakraCard,
+  ListItem,
+  Text,
+  UnorderedList,
+  Button,
+} from "@chakra-ui/react";
+import { TinaMarkdown, TinaMarkdownContent } from "tinacms/dist/rich-text";
+
+interface CardBackProps {
+  animation: string;
+  isFlipped: boolean;
+  setIsFlipped: React.Dispatch<React.SetStateAction<boolean>>;
+  name?: string | null;
+  description?: TinaMarkdownContent | TinaMarkdownContent[];
+}
+
+const CardBack: React.FC<CardBackProps> = ({
+  animation,
+  isFlipped,
+  setIsFlipped,
+  name,
+  description,
+}) => {
+  const handleClick = () => {
+    setIsFlipped(!isFlipped);
+  };
+
+  return (
+    <ChakraCard size="lg" variant="back" align="center" animation={animation}>
```

Comment:
I think the cards look a bit too big. Perhaps the size should be changed to "md"

Reply:
I agree that they're a bit large even though I already decreased the sizing drastically compared to Figma. I'll have to define size "md" first as I changed ChakraUI Card theme specifically for these cards. Maybe I'll do size "sm" as well to have it ready for tablet/mobile